changelog.d/484.misc

```diff
@@ -0,0 +1 @@
+Upgrade dependencies with no affecting breaking changes
```

Comment:
```suggestion
Upgrade dependencies
```

I wouldn't include the breaking changes line, we are expected to uphold that promise implicitly unless we bump a major version :)

Reply:
ok

Reply:
Missed to change this before merging:
Changed in 944033902033c9f7a516afc9aa5b34cfeb89e640.